components/more_direct_channels/last_post_at.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {FunctionComponent} from 'react';
+
+import Timestamp from 'components/timestamp/timestamp';
+type Props = {
+    lastPostAt?: number;
+}
+
+export const LastPostAt: FunctionComponent<Props> = ({lastPostAt}: Props): JSX.Element | null => {
+    if (!lastPostAt) {
+        return null;
+    }
+
+    return (
+        <Timestamp
+            value={lastPostAt}
+            useTime={false}
+            style='long' //eslint-disable-line react/style-prop-object
```

Comment:
Instead of overwriting the style, can we add the value in the Props? 
 ```
       // fixed
        year: 'numeric',
        month: 'long',
        day: '2-digit',
        weekday: 'long',
        hour: 'numeric',
        minute: 'numeric',
        second: 'numeric',
        hourCycle: 'h12',
        timeZoneName: 'short',
        lastPostAt: 'long',
```

Reply:
I believe this prop is specific to the component. Not sure if aliasing a feature name is fine. Also this gives us the flexibility to use `<LastPostAt />` reusable elsewhere with the `short` style.  Kindly let me know WDYT.

Reply:
I would at least name the property 'format' and make an enum for the format so it's easier for others to understand what values are possible. Could be an enum or string alternative type. If we want the flexibility to reuse <LastPostAt /> then we should add the prop to that component as well and use the same enum. You think this makes sense?

Reply:
`style` (along with `numeric`) is a member of [Intl.RelativeTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/RelativeTimeFormat) that `<Timestamp/>` makes use of.

In another PR, I'm disabling `react/style-prop-object` enforcement for `<Timestamp/>` and setting `style`'s default to 'long', so this `//eslint-disable-line` is just temporary.